src/app/ubs/ubs-admin/components/ubs-admin-order-details-form/ubs-admin-order-details-form.component.ts

```diff
@@ -289,4 +289,14 @@ export class UbsAdminOrderDetailsFormComponent implements OnInit, OnChanges {
     this.writeoffAtStationSum = +e.target.value;
     this.calculateFinalSum();
   }
+
+  shortenCommentLenght(): string {
+    const comment = (document.querySelector('#customer-comment') as HTMLInputElement)?.value;
+    if (comment && comment.length < 105) {
+      return comment;
+    }
+    if (comment && comment.length >= 105) {
+      return comment.slice(0, 105);
+    }
+  }
```

Comment:
1)  Don't use querySelector, we have formControlName for textarea (customerComment)
2)  in this case, it is correct to use the ternary operator, but the comment may be absent too 
3) I don't understand why this method is needed))

Reply:
I used optional chaining when comment value is null is becomes underfined. And then logical 'and' which means if value is underfined which is boolean false it will skip if operator. 
This method is needed because comment has particular size, and when it is too long not all text will be visible.

Reply:
Need to change style - height of field, not logic

Reply:
I changed textarea size instead of logic